andela/ah-infinity-stones-frontend
# #161966863 Install and Setup Redux

[![Build Status](https://travis-ci.org/andela/ah-infinity-stones-frontend.svg?branch=ch-setup-redux-161966863)](https://travis-ci.org/andela/ah-infinity-stones-frontend)
#### What does this PR do?
- Setup Redux

#### Description of Task to be completed?
- Install redux, react-redux and redux-saga
- Create appropriate redux folder structure
- Create sample component and apply middleware sagas.

#### How should this be manually tested?
1. `$ git clone -b ch-setup-redux-161966863 https://github.com/andela/ah-infinity-stones-frontend.git`  and `$ cd ah-infinity-stones-frontend`
2. Install the dependencies - `$ npm install`
3. Start the app by running `$ npm start` - This will run the app in the development mode.  Open http://localhost:3000 to view it in the browser.
The page will reload if you make edits. 	You will also see any lint errors in the console.
4. Test the app by running `$ npm test` - This will Launch the test runner in the interactive watch mode.

#### What are the relevant pivotal tracker stories?
[#161966863](https://www.pivotaltracker.com/story/show/161966863)

#### Screenshots
<img width="866" alt="screenshot 2019-01-09 at 15 14 01" src="https://user-images.githubusercontent.com/12385482/50898785-3eb47180-1421-11e9-8114-d645286b7235.png">
